src/content/en/updates/2017/02/navigation-preload.md

```diff
@@ -0,0 +1,338 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Navigation preload lets you overcome service worker startup time by making requests in parallel.
```

Comment:
Hmm, it doesn't improve it, it makes it not matter. Is there a better word than "overcome"?